index.html

```diff
@@ -3,11 +3,304 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-    <link rel="stylesheet" href="style.css" />
-    <title>Exclusive</title>
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Home page</title>
   </head>
   <body>
-    <h1>Exclusive</h1>
+    <main class="home-page">
+      <!-- First section -->
+      <section class="img-container">
+        <img src="images/ads/ad-phone.svg" alt="Advertising image" />
+        <img src="images/ads/ad-home.svg" alt="Advertising image" />
```

Comment:
Creo que podrías mejorar el alt de estas images.


```suggestion
        <img src="images/ads/ad-phone.svg" alt="IPhone 14 Series. Up to 10% off voucher. Shop now" />
        <img src="images/ads/ad-home.svg" alt="Sale. Get up to 30% off all home products. Buy now" />
```

Recuerda que alt es usado por los lectores de pantalla. Si la imagen tiene texto, sería ideal que el alt tenga el texto o una descripción de la imagen. También, en ambas imagenes parece haber links y boton para 'comprar ahora'. Ya que estas son imagenes, una buena idea sería encerrar cada imagen en un link 


```
  <a href="#"><img src="..." alt="..." /></a>
  <a href="#"><img src="..." alt="..." /></a>
```